airbyte-integrations/connectors/source-mysql/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcPartitionFactory.kt

```diff
@@ -294,6 +296,10 @@ class MysqlJdbcPartitionFactory(
                     LeafAirbyteSchemaType.NUMBER -> {
                         Jsons.valueToTree(stateValue?.toDouble())
                     }
+                    LeafAirbyteSchemaType.BINARY -> {
```

Comment:
Ideally I would like to move this functionality into the codecs we already have but it's a bigger change as they are built around `ResultSet`.
But better have it in implemented in one place